app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,14 @@
+class MerchantsController < ApplicationController
+  def index
+    @merchants = MerchantFacade.merchants
+  end
+
+  def show
+    @name = MerchantFacade.find_merchant_name(params[:id])
+    @items = MerchantFacade.merchant_items(params[:id])
+  end
```

Comment:
Nice job here. You could also consider having a MerchantItemsFacade, and you could create MerchantItem objects to send to the view -- this could limit how much you're sending to the view (from two variables to two)